RFS/docker/TestSource_ES_7_10/test-resources/inventory.md

```diff
@@ -0,0 +1,45 @@
+## Test Resources
```

Comment:
Thanks for the sample and discussion. Wasn't sure this would be possible within the stage of a Dockerfile but do prefer the reproducibility of this and eliminating this as a pattern that others might follow to store larger datasets. Have updated to use a multi-stage Docker approach instead now